.ci/.e2e-platforms.yaml

```diff
@@ -40,3 +40,8 @@ PLATFORMS:
     image: "ami-0d90bed76900e679a"
     instance_type: "c5.4xlarge"
     username: admin
+  macos12:
+    description: "Mac OS 12 platform"
+    image: "ami-094dfaad20f351cb7"
+    instance_type: "c5.4xlarge"
```

Comment:
@jlind23 , I've found some references in [here](https://aws.amazon.com/premiumsupport/knowledge-center/ec2-instance-types-disabled-at-launch/)

> The instance types A1, T4g, M6g, C6g, and R6g use AWS Graviton processors. AWS Graviton processors use the aarch64 (ARM64) architecture instead of x86_64 (AMD64).


```suggestion
    instance_type: "c6g.4xlarge"
```

<img width="1160" alt="image" src="https://user-images.githubusercontent.com/2871786/165782781-29f28ff8-592e-4764-98f8-d3c96d8a8d1c.png">

Reply:
The `c5.4xlarge` uses similar resources

<img width="1187" alt="image" src="https://user-images.githubusercontent.com/2871786/165783156-ba95903f-178a-4861-b926-a9eed045e9ea.png">